src/Creditor.sol

```diff
@@ -90,5 +90,5 @@ abstract contract Creditor {
      * @notice Starts the liquidation of an account and returns the open position of the Account.
      * @return openPosition the open position of the Account
```

Comment:
`* @param liquidationInitiator The address that starts the liquidation flow.`

Reply:
+ in LendingPool.sol the variable is named "initiator", in accounts also liquidationInitiator